files/zh-cn/web/svg/element/discard/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: <discard>
+slug: Web/SVG/Element/discard
+tags:
+  - Element
+  - NeedsExample
+  - Reference
+  - SVG
+  - SVG Animation
+browser-compat: svg.elements.discard
```

Comment:
just remove it, it is used by `{{Specifications}}` and `{{Compat}}` macros, and could just fallback to `en-US`.

```suggestion
```